src/frontend/src/btc/services/btc-send.services.ts

```diff
@@ -9,8 +9,7 @@ import { mapToSignerBitcoinNetwork } from '$lib/utils/network.utils';
 import { waitAndTriggerWallet } from '$lib/utils/wallet.utils';
 import type { Identity } from '@dfinity/agent';
 import type { BitcoinNetwork } from '@dfinity/ckbtc';
-import { toNullable } from '@dfinity/utils';
-import { hexStringToUint8Array } from '@dfinity/utils/dist/types/utils/arrays.utils';
```

Comment:
@AntonioVentilii-DFINITY is it feasable to have an eslint rule for this you think?